cmd/build-packages.go

```diff
@@ -25,6 +28,8 @@ compiled once.
 `,
 	RunE: func(cmd *cobra.Command, args []string) error {
 
+		flagBuildPackagesRoles := viper.GetString("roles")
```

Comment:
I know this works (at least, I've tested it), but I have no idea _how_.  This is talking to the global viper, not the custom one on line 55...